### PR TITLE
reorganization features

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: trello/api/features/Card.feature [hunk 1]
```diff
@@ -361,3 +500,45 @@ Feature: Card
...
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
+
+
+  @Negative
+  Scenario: It can not get the member of a card that doesn't exist
+    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/members"
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Card.feature]: add a new line please

File: trello/api/features/List.feature [hunk 2]
```diff
@@ -1,5 +1,62 @@
-Feature: List
-  Manages list of the board
+Feature: Board's list
+  Manages lists of the board
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/List.feature]: add "As a regular user..."

File: trello/api/features/List.feature [hunk 3]
```diff
@@ -209,3 +266,56 @@ Feature: List
...
+    And Should return status code 200
+    And Saves response as "ListObject"
+    When Sets a "PUT" request to "/lists/ListObject.id/softLimit"
+      | key   | value |
+      | value | 5001  |
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid value for value"
+
+
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/List.feature]: delete this lines

File: trello/api/features/Organization.feature [hunk 4]
```diff
@@ -1,7 +1,52 @@
...
+    And Saves response as "organization_membersObject"
+    And Validates response body with
+      | key                      | value      |
+      | memberships.1.memberType | normal     |
+      | members.1.fullName       | Angel Owen |
+    And Validates schema with "put_organization_members.json"
+    And Sets a "GET" request to "/organizations/organization_membersObject.id/members"
+    And Sends request
+    And Should return status code 200
+
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Organization.feature:48]: add a new line here

File: trello/api/features/Card.feature [hunk 5]
```diff
@@ -361,3 +500,45 @@ Feature: Card
       | closed | False    |
       | name   | findCard |
     And Validates schema with "get_card_schema.json"
+
+      @Negative
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Card.feature]: check identation

File: trello/api/features/Card.feature [hunk 6]
```diff
@@ -361,3 +500,45 @@ Feature: Card
       | closed | False    |
       | name   | findCard |
     And Validates schema with "get_card_schema.json"
+
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Card.feature:503]: add a new line here please

File: trello/api/features/Card.feature [hunk 7]
```diff
@@ -1,7 +1,146 @@
...
+      | key        | value       |
+      | name       | myChecklist |
+      | checkItems | []          |
+    And Validates schema with "checklist_schema.json"
+    And Sets a "GET" request to "/cards/CardObject.id"
+    And Sends request
+    And Should return status code 200
+
+
+      @Functional
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Card.feature]: check identation please

File: trello/api/features/Card.feature [hunk 8]
```diff
@@ -1,7 +1,146 @@
...
+      | idList | (ListObject.id) |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "CardObject"
+    When Sets a "POST" request to "/cards/CardObject.id/checklists"
+      | key  | value       |
+      | name | myChecklist |
+    And Sends request
+    And Should return status code 200
+    Then Saves response as "CardObjectUpdate"
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Card.feature]: invert **Then** and **And**

File: trello/api/features/List.feature [hunk 9]
```diff
@@ -1,5 +1,61 @@
-Feature: List
-  Manages list of the board
+Feature: Board's list
+ As a regular user, it wants to manage a list, and creates a list.
+  @Smoke
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/List.feature:4]: add a new line above this one, please

File: trello/api/features/List.feature [hunk 10]
```diff
@@ -209,3 +265,54 @@ Feature: List
     And Sets a "GET" request to "/lists/List_Object.id"
     And Sends request
     And Should return status code 200
+
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/List.feature:269]: add a new line here please

File: trello/api/features/Organization.feature [hunk 11]
```diff
@@ -1,7 +1,53 @@
 Feature: Organization
   As a regular user, it wants to manage a organization, and creates a organization.
 
-  @Functional
+    @Smoke
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/Organization.feature]: check identation please

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**